client/app/telecom/pack/xdsl/access/pack-xdsl-access.controller.js

```diff
@@ -338,4 +349,24 @@ angular.module('managerApp').controller('XdslAccessCtrl', class XdslAccessCtrl {
       this.$scope.loaders.details = false;
     });
   }
+
+  getIncident() {
+    return this.OvhApiXdsl.Incident().v6().get({
+      serviceName: this.$stateParams.serviceName,
+    }).$promise
+      .then(() => {
+        this.hasIncidentOccured = true;
+      })
+      .catch((error) => {
+        const errorMessage = _.get(error, 'data.message', '');
+        if (errorMessage === XDSL_NO_INCIDENT_MESSAGE) {
```

Comment:
Could we rely  on checking the `error.status === 404` instead ? I don't know if that makes sense , let's suppose the API error message changes, we have to change it here also

Reply:
Yes I would prefer rely on the `404` code, but the API returns two different errors for the same error code. I asked them to change this but they won't as this is made this way